src/templates/kit/icons/fundraising.svg

```diff
@@ -0,0 +1 @@
+<svg width="114" height="114" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="114" height="114" rx="20" fill="url(#paint0_linear)"/><path d="M78.065 77.964c-11.561 0-20.934-9.372-20.934-20.933 0-11.562 9.373-20.934 20.934-20.934C89.627 36.097 99 45.47 99 57.031c0 11.56-9.373 20.933-20.935 20.933zM35.935 77.964C24.373 77.964 15 68.592 15 57.03c0-11.562 9.373-20.934 20.935-20.934 11.561 0 20.934 9.372 20.934 20.934 0 11.56-9.373 20.933-20.934 20.933z" fill="#fff" fill-opacity=".2" stroke="#fff" stroke-width="1.5"/><path d="M56.966 56.866c-11.562 0-20.935-9.372-20.935-20.933C36.031 24.373 45.404 15 56.966 15s20.935 9.372 20.935 20.933-9.373 20.933-20.935 20.933zM56.966 99c-11.562 0-20.935-9.371-20.935-20.932 0-11.561 9.373-20.933 20.935-20.933s20.935 9.372 20.935 20.933C77.9 89.628 68.528 99 56.966 99z" fill="#fff" fill-opacity=".2" stroke="#fff" stroke-width="1.5"/><defs><linearGradient id="paint0_linear" x1="-73.807" y1="-59.511" x2="107.304" y2="120.726" gradientUnits="userSpaceOnUse"><stop stop-color="#FF9372"/><stop offset="1" stop-color="#FFD770"/></linearGradient></defs></svg>
```

Comment:
We may want to update this icon to be 24x24 (it is currently 114x114) to align with our other icons, but it's not a big deal since it's an svg.